Update README.md (#6)
vtex-apps/order-profile: docs/README.md

> 📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

Reviewer: ```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/order-profile) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

> # ORDER PROFILE

Reviewer: ```suggestion
# Order Profile
```

> ⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.** ⚠️

Reviewer: ```suggestion
> :warning: *Although ready to be installed, the Order Profile app is currently an **open-beta project**. Bear in mind that due to that status, you can expect an accelerated state of evolution.*
```

<!-- DOCS-IGNORE:start -->
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
<!-- DOCS-IGNORE:end -->

> Centralizes all profile related requests to the Checkout API

Reviewer: The Order Profile app is responsible for storing all user profiles that perform requests to the [VTEX Checkout API](https://developers.vtex.com/vtex-developer-docs/reference/checkout-api-overview).

## Configuration 

> TBC

Reviewer: 👀

> TBC

Reviewer: ```suggestion
🚧 To be defined.
```

## Customization

> TBC

Reviewer: 👀

> TBC

Reviewer: ```suggestion
🚧 To be defined.
```
